packages/E2ETest/windows/ReactUWPTestApp/ReactUWPTestApp.csproj

```diff
@@ -20,7 +20,6 @@
     <FileAlignment>512</FileAlignment>
     <ProjectTypeGuids>{A5A43C5B-DE2A-4C0C-9213-0A381AF9435A};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
-    <PackageCertificateThumbprint>8FA11EA9E8951B266F245AFE92B3060CECAF4503</PackageCertificateThumbprint>
```

Comment:
@asklar Making sure should indeed be removed

Reply:
this seems fine to remove